djangocms_alias/admin.py

```diff
@@ -38,7 +38,7 @@
     alias_content_admin_classes.insert(0, ExtendedVersionAdminMixin)
     alias_content_admin_list_display = ('name', 'get_category',)
     alias_content_admin_list_filter = (SiteFilter, LanguageFilter, UnpublishedFilter)
```

Comment:
`alias_content_admin_list_filter` appears to be duplicated historically, we should clean that.